config/routes.rb

```diff
@@ -777,8 +777,10 @@
       resources :organisations, only: [] do
         get '/' => 'organisation_permissions#organisations', on: :collection
         resources :organisation_permissions, path: '/organisation-permissions', only: %i[index edit update]
-        resources :users, path: '/users', only: %i[index show edit destroy] do
+        resources :users, path: '/users', only: %i[index show edit update destroy] do
           member do
+            get :check
+            put :commit
```

Comment:
Why is this a put instead of a post?

Reply:
POST really means: "Here's some data, create a new thing with it". So I can POST to /thing and I'd expect a will new thing to be created with the data I send.

PUT means: "Here's some data, update the specified thing with it". So I can PUT to /thing/123 and I'd expect thing number 123 to be updated with the data I send.

(Bonus: there's PATCH which is similar to PUT, but I'd expect to be able to send partial data, like { "first_name": "Ben" }, and have the resource be updated with only the fields I send. PUT really expects a full description of the resource, even if no changes are being made to some fields. E.g to just change my name via a PUT, I'd have to send { "first_name": "Ben", "age": 21, "favourite_fruit": "melon" } even though I'm not changing my favourite fruit.)

Reply:
Maybe we should change these routes to act on /users/:id/permissions, and call it PUT